lib/vendors/axis-communications/sv_vendor_axis_communications.c

```diff
@@ -21,6 +21,7 @@
 #include "includes/sv_vendor_axis_communications.h"
 
 #include <stdbool.h>
+#include <stdio.h>
```

Comment:
What in this patch requires this include(stdio.h)?

Reply:
Nothing. It has been removed locally.